ci-operator/config/openshift/ib-sriov-cni/openshift-ib-sriov-cni-master__okd-scos.yaml

```
@@ -0,0 +1,46 @@
+base_images:
+  ocp_builder_rhel-8-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.21-openshift-4.16
+  ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-openshift-4.16
+  origin_scos-4.15_base-stream9:
+    name: scos-4.15
+    namespace: origin
+    tag: base-stream9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.rhel7
+  from: origin_scos-4.15_base-stream9
+  inputs:
+    ocp_builder_rhel-8-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-4.16
+    ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+  to: sriov-infiniband-cni
+promotion:
+  to:
+  - name: "4.16"
```

```suggestion
  - name: "scos-4.16"
```
For historical reasons, `origin/4.16` is the OKD/FCOS ImageStream (which we currently don't populate) and `origin/scos-4.16` is the OKD/SCOS this is supposed to get promoted to.

This needs to change here and elsewhere in this PR.

---

Hi @LorbusChris I try that but it removes the image from the image_mirror that is all the point of what I need

```
-registry.ci.openshift.org/origin/4.16:metallb quay.io/openshift/origin-metallb:4.16 quay.io/openshift/origin-metallb:4.16.0 quay.io/openshift/origin-metallb:latest
-registry.ci.openshift.org/origin/4.16:metallb-operator quay.io/openshift/origin-metallb-operator:4.16 quay.io/openshift/origin-metallb-operator:4.16.0 quay.io/openshift/origin-metallb-operator:latest
-registry.ci.openshift.org/origin/4.16:metallb-operator-bundle quay.io/openshift/origin-metallb-operator-bundle:4.16 quay.io/openshift/origin-metallb-operator-bundle:4.16.0 quay.io/openshift/origin-metallb-operator-bundle:latest
 registry.ci.openshift.org/origin/4.16:okd-rpms quay.io/openshift/origin-okd-rpms:4.16 quay.io/openshift/origin-okd-rpms:4.16.0 quay.io/openshift/origin-okd-rpms:latest
 registry.ci.openshift.org/origin/4.16:openstack-installer quay.io/openshift/origin-openstack-installer:4.16 quay.io/openshift/origin-openstack-installer:4.16.0 quay.io/openshift/origin-openstack-installer:latest
 registry.ci.openshift.org/origin/4.16:operator-marketplace quay.io/openshift/origin-operator-marketplace:4.16 quay.io/openshift/origin-operator-marketplace:4.16.0 quay.io/openshift/origin-operator-marketplace:latest
 registry.ci.openshift.org/origin/4.16:ovirt-installer quay.io/openshift/origin-ovirt-installer:4.16 quay.io/openshift/origin-ovirt-installer:4.16.0 quay.io/openshift/origin-ovirt-installer:latest
-registry.ci.openshift.org/origin/4.16:sriov-cni quay.io/openshift/origin-sriov-cni:4.16 quay.io/openshift/origin-sriov-cni:4.16.0 quay.io/openshift/origin-sriov-cni:latest
-registry.ci.openshift.org/origin/4.16:sriov-dp-admission-controller quay.io/openshift/origin-sriov-dp-admission-controller:4.16 quay.io/openshift/origin-sriov-dp-admission-controller:4.16.0 quay.io/openshift/origin-sriov-dp-admission-controller:latest
-registry.ci.openshift.org/origin/4.16:sriov-infiniband-cni quay.io/openshift/origin-sriov-infiniband-cni:4.16 quay.io/openshift/origin-sriov-infiniband-cni:4.16.0 quay.io/openshift/origin-sriov-infiniband-cni:latest
-registry.ci.openshift.org/origin/4.16:sriov-network-config-daemon quay.io/openshift/origin-sriov-network-config-daemon:4.16 quay.io/openshift/origin-sriov-network-config-daemon:4.16.0 quay.io/openshift/origin-sriov-network-config-daemon:latest
-registry.ci.openshift.org/origin/4.16:sriov-network-device-plugin quay.io/openshift/origin-sriov-network-device-plugin:4.16 quay.io/openshift/origin-sriov-network-device-plugin:4.16.0 quay.io/openshift/origin-sriov-network-device-plugin:latest
-registry.ci.openshift.org/origin/4.16:sriov-network-metrics-exporter quay.io/openshift/origin-sriov-network-metrics-exporter:4.16 quay.io/openshift/origin-sriov-network-metrics-exporter:4.16.0 quay.io/openshift/origin-sriov-network-metrics-exporter:latest
-registry.ci.openshift.org/origin/4.16:sriov-network-operator quay.io/openshift/origin-sriov-network-operator:4.16 quay.io/openshift/origin-sriov-network-operator:4.16.0 quay.io/openshift/origin-sriov-network-operator:latest
-registry.ci.openshift.org/origin/4.16:sriov-network-webhook quay.io/openshift/origin-sriov-network-webhook:4.16 quay.io/openshift/origin-sriov-network-webhook:4.16.0 quay.io/openshift/origin-sriov-network-webhook:latest

```